src/evd/cyclic/evd_cyclic_blocked_vectorize.cpp

```diff
@@ -159,8 +160,9 @@ void evd_cyclic_blocked_less_copy_vectorize(struct matrix_t Data_matr, struct ma
                 copy_block(Amat, j_block, i_block, Ablockmat, 1, 0, block_size);
                 copy_block(Amat, j_block, j_block, Ablockmat, 1, 1, block_size);
 
-                // evd_block_vector(Ablockmat, Vblockmat);
-                evd_subprocedure_vectorized(Ablockmat, Vblockmat);
+                evd_block_vector(Ablockmat, Vblockmat);
+                //We have the wrong cost for this.
+                //evd_subprocedure_vectorized(Ablockmat, Vblockmat);
```

Comment:
No, we have to use it because it is the best performing version. We need to fix its cost, but we can't discard it!

Reply:
For this PR, I only want master to give reasonable results for performance and roofline. So I commented it for now. There are other small issues with the blocking as well, but I think that should be another PR.